blogposts/2021/dev-tool-time-seth-vargo.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Dev Tool Time with Seth Vargo - April 14, 2021
+description: On April 14, 2021, Dev Tool Time welcomed Seth Vargo as our first guest. Seth showed us his productivity optimized desk setup, talked about great window management and keyboard shortcuts. He shared his .gitconfig and much more.
```

Comment:
Check out the recording of our first episode of Dev Tool Time, in which Google Cloud Engineer Seth Vargo shares his tips for a productivity-optimized desk setup, efficient window management, and keyboard shortcuts.

Reply:
```suggestion
description: Check out the recording of our first episode of Dev Tool Time, in which Google Cloud Engineer Seth Vargo shares his tips for a productivity-optimized desk setup, efficient window management, and keyboard shortcuts.
```